policy/lib/tekton/task.rego

```diff
@@ -173,6 +173,13 @@ build_tasks(attestation) := [task |
 	count(image_digest) > 0
 ]
 
+# return the tasks that have "TEST_OUTPUT" as a result
+tasks_test_output(attestation) := [task |
```

Comment:
NIT: `test_tasks` ? We don't need to encode what it means to be a test task in the name.